releases/v19.2.0-rc.2.md

```diff
@@ -0,0 +1,102 @@
+---
+title: What&#39;s New in v19.2.0-rc.2
+toc: true
+summary: Additions and changes in CockroachDB version v19.2.0-rc.2 since version v19.2.0-rc.1
+---
+
+## October 28, 2019
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-rc.2.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-rc.2.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-rc.2.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v19.2.0-rc.2
+~~~
+
+### General changes
+
+- Go 1.12.10 is now required to build CockroachDB from source. [#41647][#41647] {% comment %}doc{% endcomment %}
+- Named ports in URLs are no longer supported due to a [URL parsing security fix in Go 1.12.8](https://github.com/golang/go/commit/3226f2d492963d361af9dfc6714ef141ba606713). Use numeric port numbers instead. [#41647][#41647] {% comment %}doc{% endcomment %}
+
+### SQL language changes
+
+- Mutations under [`UNION` or `UNION ALL`](../v19.2/selection-queries.html#union-combine-two-queries) are once again supported. [#41500][#41500] {% comment %}doc{% endcomment %}
+- Added new `skip` and `nullif` options for [`IMPORT ... DELIMITED DATA`](../v19.2/import.html), which ignore the specified number of lines at the beginning of the file and force all empty string fields to be inserted as `NULL` in the imported table, respectively. [#41656][#41656] {% comment %}doc{% endcomment %}
+- It is no longer possible to use [`IMPORT INTO`](../v19.2/import-into.html) with an interleaved table. This restriction is temporary and will be lifted in a future release. [#41651][#41651] {% comment %}doc{% endcomment %}
```

Comment:
"interleaved table" could be a link to https://www.cockroachlabs.com/docs/stable/interleave-in-parent.html to save the user a search